scripts/compare_benchmarks.py

```diff
@@ -2,15 +2,23 @@
 
 import json
 import sys
-from beautifultable import BeautifulTable
+from terminaltables import AsciiTable
 from termcolor import colored
+from scipy.stats import ttest_ind
+from array import array
```

Comment:
Do we need this because `ttest_ind()` does not handle plain python lists?

Reply:
Unfortunately yes.